README.md

```diff
@@ -171,3 +174,9 @@ In short, Buildah is an efficient way to create OCI images while Podman allows
 you to manage and maintain those images and containers in a production environment using
 familiar container cli commands.  For more details, see the
 [Container Tools Guide](https://github.com/containers/buildah/tree/master/docs/containertools).
+
+## Podman Legacy API (Varlink)
+Podman previously offered a Varlink-based API for remote management of containers.
+This API is has been deprecated by the REST API.
```

Comment:
```suggestion
However, the Varlink API support is currently in maintenance mode and will be deprecated when Podman version 2.0 is released.  It will then be removed completely in a future release.
```